apps/auction2/components/auction_grid/index.js

```diff
@@ -0,0 +1,18 @@
+import { default as React, PropTypes } from 'react';
+import AuctionGridArtwork from '../auction_grid_artwork/index'
+
+export default function AuctionGrid({ artworks }, _) {
```

Comment:
👍 works for me! When @orta adds Typescript to Force we can revisit the need for `propTypes` 😛